smithyctl/command/component/package.go

```diff
@@ -0,0 +1,190 @@
+package component
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/go-errors/errors"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+
```

Comment:
nit: extra empty line here

Reply:
`make fmt` keeps adding the line 🤔 

Do you mind if we ignore this for now?

I think we need better support for `go imports` regardless